src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2289,6 +2327,81 @@ components:
             example: 
             - "path/to/file1"
             - "path/to/file2"
+    JobQueue:
+      description: Structure of jobs queue
+      type: object
+      properties:
+        jq_pk:
+          description: Job queue id
+          type: string
+        jq_job_fk:
+          description: Job id
+          type: string
+        jq_type:
+          description: Job queue type
+          type: string
```

Comment:
# PLEASE DO NOT EXPOSE DB INTERNALS
in the REST endpoints!
Please change the names of the parameters at the minimum to be consistent with other REST endpoints.

Reply:
Sure, I'll do that!